docs/installguide/install_all.rst

```diff
@@ -94,49 +101,35 @@ The file may be named as if it was intended for Ubuntu but works just as well fo
 Debian, Raspberry Pi, Linux Mint etc.
 
 You will be prompted to enter some configuration information.
-You should read the on-screen instructions carefully, but some explanation is included here:
-
-1. Choose weather you want to run KA Lite on boot or not. We recommend choosing yes, as it simplifies data management.
-If you choose not to, you must manually start KA lite every time.
-
-.. note::
-    Running KA Lite as different users creates different sets of data files, so it's recommended that you run KA Lite as the same user every time.
-
-.. image:: linux-install-startup.png
-  :class: screenshot
-
-2. If you chose to start on boot in the previous step, you will be prompted to choose the owner for the KA Lite server
-process. Generally the default value is ok.
-
-.. image:: linux-install-owner.png
-  :class: screenshot
-
-3. You will be asked to review your choices, and finally KA Lite will start automatically when installation is complete.
+You should read the on-screen instructions carefully.
 
+Raspberry Pi
+============
 
-.. tip::
-    If you want to receive automatic updates from online sources, you can
-    also use :ref:`ppa-installation`.

```

Comment:
I don't think removing the reference to PPA installations is a good idea if that's the intention?